tests/security/yast2_apparmor/settings_disable_enable_apparmor.pm

```diff
@@ -47,6 +47,7 @@ sub run {
     send_key "alt-e";
     assert_screen("AppArmor-Settings-Enable-Apparmor");
     wait_screen_change { send_key "alt-q" };
+    type_string("\n");
```

Comment:
@lilyeyes why you type a "\n" here,  a workaround?

Reply:
> @lilyeyes why you type a "\n" here, a workaround?

I thing it can make the cursor appears again (sometime the cursor disappears for no reason).
But it is hard to verify as this issue random.

Reply:
Got it, it makes sense, then I would suggest you add some comments here

Reply:
same as mine confusion, is "\n" a common usage?

Reply:
> same as mine confusion, is "\n" a common usage?

I don't know.